### DISCUSSION TITLE
Ghost unread notification

### DISCUSSION THREAD

**Discussion by katopz** [community/community#42762] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Bug

### Body

I always have 1 notification remain.

<img width="994" alt="image" src="https://user-images.githubusercontent.com/97060/209769198-a7edc191-a0b7-4350-b84f-c6dffea82b02.png">

How to get rid of this?

<img width="990" alt="image" src="https://user-images.githubusercontent.com/97060/209769327-59afa44a-e5ad-47cc-ab39-e69e2f3b2d50.png">

Inbox also didn't list that ghost item to mark as read.

**Comment by radar-blip** [comment, marked as answer]
Lots of people had luck with that trying the things in these discussions: #6874

Or here: #32712
